src/util/StringStream.js

```diff
@@ -73,9 +74,14 @@ class StringStream {
   current(){return this.string.slice(this.start, this.pos)}
   hideFirstChars(n, inner) {
```

Comment:
It's just not visible in the diff, there is still one line left using lineStart:
sol() {return this.pos == this.lineStart}
There is only one more mode using hideFirstChars, but it doesn't have any test cases so I'm not quite sure how just changing "sol() {return this.pos == this.lineStart}" to "sol() {return this.pos == 0}" would impact that mode, so I added hideIndentation.